README.md

```diff
@@ -48,6 +48,23 @@ First-timers-bot is built with [Probot](https://probot.github.io/).
 
 <p align="center"><img src="/assets/Issue-Done.png"></p>
 
+### Configuration
+
+If you would like to add your own configuration, add a file called 'first-timers.yml' to the .github directory of the repository where the app is installed with the following options. 
```

Comment:
🤔 I think we can make this a bit more clear. It sounds like the app is not installed into the .github directory, but that’s not the case.

Maybe say something shorter like this?

> The `first-timers` app works without configuration. If you want to change the default settings, create a `.github/first-timers.yml` file with the content below can adjust the options to your preference